modules/math/src/utils/assert.ts

```diff
@@ -1,7 +1,7 @@
 // Replacement for the external assert method to reduce bundle size
 // Note: We don't use the second "message" argument in calling code,
 // so no need to support it here
-export function assert(condition, message) {
+export function assert(condition: Boolean, message?: string): void {
```

Comment:
typescript uses lower case `boolean`
```suggestion
export function assert(condition: boolean, message?: string): void {
```